src/mempool.cpp

```diff
@@ -41,13 +38,19 @@ void MemPool::uploadVbkContext(const VTB& vtb) {
   }
 }
 
+void MemPool::uploadVbkContext(const ATV& atv) {
+  block_index_[atv.containingBlock.getHash()
+                   .trimLE<vbk_prev_block_hash_size>()] = atv.containingBlock;
+
+  for (const auto& b : atv.context) {
+    block_index_[b.getHash().trimLE<vbk_prev_block_hash_size>()] = b;
```

Comment:
instead of using trim, define a function `short_hash_t makeShortHash(hash_t)`

Reply:
fixed.